INSTALL.md

```diff
@@ -1773,9 +1773,9 @@ the error message. It might be that the compiler is already telling you
 exactly what you need to do to fix your problem.
 
 There may be reasons for the failure that aren't problems in OpenSSL itself,
-for example if the compiler reports missing standard or third party headers.
+for example, if the compiler reports missing standard or third party headers.
 
-If the build succeeded previously, but fails after a source or configuration
+If the build succeeded previously but fails after a source or configuration
```

Comment:
`but fails after a source or configuration change` seems to be a separate clause, so don't we need the comma here?